docs/components/layout-navigation/tabs.md

```diff
@@ -118,3 +118,137 @@ import { SiTabComponent, SiTabsetComponent } from '@siemens/element-ng/tabs';
 <si-docs-api component="SiTabComponent"></si-docs-api>
 
 <si-docs-types></si-docs-types>
+
+## Code (next) ---
+
+Element implements tabset and tab components that provides the correct styling and responsive behavior.
+It supports the usage combined with the Angular router (preferred way) or alternatively as plain components.
+
+### Tabs with Angular Router
+
+When using the tabs with the Angular router, each tab is just an anchor with a `routerLink`.
+Make sure to nest the `<router-outlet />` inside the `si-tabset-next` component as the tabset applies some accessibility attributes.
+No need to apply `routerLinkActive`. 
+This is done automatically.
```

Comment:
Took me a while to understand what you meant here, IMO it's part of the same sentence.

```suggestion
No need to apply `routerLinkActive`, this is done automatically.
```